apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -1,6 +1,6 @@
 You can use Fluent UI's icons in a few ways, depending on if you're using Fluent UI React or Fabric Core.
 
-### Fluent UI React
+### Fluent UI React: Font-based Icons
```

Comment:
This should use title case for "icons" (I forget whether "font" should also be lowercase or not...)
```suggestion
### Fluent UI React: Font-based icons
```